.github/workflows/prepare-for-staging-deploy.yml

```diff
@@ -113,6 +113,7 @@ jobs:
             git checkout -b $BRANCH_NAME
 
             cd helm-charts/shared-services/per-az/environment/staging
```

Comment:
where's TF_IMAGE_REPOSITORY variable defined ?

Reply:
Defined it above now, in create pull request step `env`